### DISCUSSION TITLE
Welcome to open-nic-driver Discussions!

### DISCUSSION THREAD

**Discussion by aidanr-xlnx** [Xilinx/open-nic-driver#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by attdone** [comment]
Hi,
Whether the latest open-nic-driver is default set for sn1000 device? 
I wanted the driver for Alveo u200, so what changes in the driver to be made?
It is required to enable  XMC_OPENNIC and other undefined Macros of XMC; to support Alveo u200?

**Comment by cneely-amd** [comment]
Hi @attdone ,

The open-nic-driver can be used across the various Alveo cards, without changes, although a lot of the changes made a few months ago were for adding support to the SN1022/Alveo U45N.

If you want a version without xmc, maybe try an earlier version before the bulk of the changes for SN1022 (this is the commit with most of the SN1022 changes):
https://github.com/Xilinx/open-nic-driver/commit/4d549436d09c58d24035665c044c0c60e18af1db

So, if you go to any versions before that, the driver won't have XMC_OPENNIC-related code.

--Chris

**Comment by attdone** [comment]
Hi @cneely-amd
Thanks for the update.
Please let me know, what changes to be made to support for u200 in latest source, which should include XMC_OPENNIC related codes?
